meson.build

```diff
@@ -394,17 +402,14 @@ if sys_windows or get_option('lua-interpreter') == 'lua'
       break
     endif
   endforeach
+  if not lua.found()
+    lua = dependency('lua', fallback : ['lua', 'lua_dep'])
+    if not lua.found()
+      error('Lua not found')
+    endif
+  endif
 else
   lua = dependency(get_option('lua-interpreter'))
-  lua_pc_name = 'luajit'
```

Comment:
Are there any other options for lua other than `lua` and `luajit`?

Reply:
Currently `lua-interpreter` can be `luajit` (default) or `lua`, and then `lua_pc_name` can be `luajit`, `lua`, `lua51`, `lua-5.1`, `lua5.1`, `lua52`,  `lua-5.2` or `lua5.2'`.
I haven't tried to set `lua-interpreter` to something different from what is specified at `meson_options.txt` but I expect meson to prevent that, so if that is the case we could remove the else part completly.

Reply:
I think we can remove this then.